go.mod

```
@@ -1,6 +1,6 @@
 go 1.21
 
-module github.com/cosmos/cosmos-sdk
+module github.com/wr1159/cosmos-sdk
```

```suggestion
module github.com/cosmos/cosmos-sdk
```
Make change before merge